packages/suite/src/views/settings/general/SettingsGeneral.tsx

```diff
@@ -19,18 +19,23 @@ import { ShowLog } from './ShowLog';
 import { ClearStorage } from './ClearStorage';
 import { VersionWithUpdate } from './VersionWithUpdate';
 import { EarlyAccess } from './EarlyAccess';
+import { BitcoinUnits } from './BitcoinUnits';
 
 export const SettingsGeneral = () => {
-    const { desktopUpdate, tor } = useSelector(state => ({
+    const { desktopUpdate, tor, accounts } = useSelector(state => ({
         desktopUpdate: state.desktopUpdate,
         tor: state.suite.tor,
+        accounts: state.wallet.accounts,
     }));
 
+    const hasBtc = !!accounts?.find(account => account.symbol === 'btc');
```

Comment:
do we want to change units only for btc? i think it should cover all bitcoin-like coins, no?

Reply:
It was not in the initial task at least 🤷‍♂️ @hynek-jina

Reply:
We are currently dealing exclusively with bitcoin. Sats are not standard for other bitcoin-like coins.

Reply:
technically they are supported, so maybe at least testnet/regtest should be considered
![Screenshot from 2022-06-06 11-04-46](https://user-images.githubusercontent.com/3435913/172131032-496762a0-9f29-4f38-a899-dfc2b3d43a24.png)
?

Reply:
Testnet makes sense IMO

Reply:
Good point..
@dahaca could you please extend it on Bitcoin testnet/regtest?

Reply:
@dahaca , so lets define that feature in [network config](https://github.com/trezor/trezor-suite/blob/develop/packages/suite/src/config/wallet/networks.ts#L15=) then (like for example rbf)

Reply:
i also noticed that this feature is not available in [older FW versions](https://github.com/trezor/trezor-firmware/search?q=amount_unit&type=commits) so i guess this feature should be excluded for them as well? (again, same as rbf)

Reply:
i've adjusted `unavailableCapabilities` in related PR, so the approach shoul be axacly the same as in RBF (in fact this feat. was added in the same FW version as rbf)

Reply:
done

Reply:
FW support check is still missing, you are only checking network features

Reply:
b1097cad2be09fc04569fd0ecbda3f1632d2d09a